homeassistant/components/mqtt/mixins.py

```diff
@@ -766,6 +782,8 @@ async def async_tear_down(self) -> None:
 class MqttDiscoveryUpdate(Entity):
     """Mixin used to handle updated discovery message for entity based platforms."""
 
+    _integration_info: MqttIntegrationInfo | None
```

Comment:
There's no reason to store this in the entity object since it's only used for logging